_pages/speaking.html

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: Speaking at DjangoCon US
+heading: Speaking at DjangoCon US
+permalink: /speaking/
+description: Information about submitting a proposal to speak at DjangoCon US
+---
+
+<div class="content">
+  <section class="section-pad">
+    <h2>Our Call for Proposals is now open!</h2>
+    <div class="column">
+      <p>Submit your <a href="{{ site.cfp_application }}">talk or tutorial proposal</a> by <strong>July 18 <a href="https://time.is/compare/0000_18_Jul_2021_in_Anywhere_on_Earth">AoE</a></strong>, and encourage your friends and colleagues to do the same.</p>
+      <p>We'll let you know by <strong>August 16</strong> if your proposal has been accepted.</p>
+      <a class="button" href="{{ site.cfp_application }}">Submit a Proposal</a>
+    </div>
+  </section>
+
+  <section class="section-pad column">
+    <div>
+      <h2>Why Speak at DjangoCon US?</h2>
+
+      <ul>
+        <li>Presenters receive a free ticket to DjangoCon US!</li>
+        <li>Professionally produced video of your talk published to our YouTube channel. (You may opt out of this.)</li>
```

Comment:
Does it count as professionally produced if they're recording their own talk?

Reply:
@drewbrew That is an excellent question -- @jefftriplett do you know if we are doing any post-processing of the videos they submit? Should I just rephrase to "Video of your talk published..."